### DISCUSSION TITLE
Study Polyoxovanadates with XTB

### DISCUSSION THREAD

**Discussion by crispom** [grimme-lab/xtb#1150] [Q&A]
I try to optimise the structure of polyoxovanadate [H4V10O28]-- with XTB.
The structure is stable at DFT  and MP2 level.  The problem it is the optimisation disrupt the structure of the ion until it has a too strange structure and the SCF procedure does not converge. All V atoms have the +5 oxidation state and the ion is diamagnetic. 
Somebody has experience with compounds of this nature or other polyxometalates? There are SCF parameters that I can stress?

**Comment by haneug** [comment]
Hi,
I can recommend the to try the procedures outlined here: https://github.com/grimme-lab/xtb/discussions/548#discussioncomment-1798217
Best,
Hagen
